src/ripple/net/impl/InfoSub.cpp

```diff
@@ -136,4 +136,17 @@ InfoSub::getRequest()
     return request_;
 }
 
+void
+InfoSub::setApiVersion(unsigned int apiVersion)
+{
+    apiVersion_ = apiVersion;
+}
+
+unsigned int
+InfoSub::getApiVersion() const noexcept
+{
+    assert(apiVersion_ > 0);
```

Comment:
Since `decltype(apiVersion_)` is an `unsigned int`, is this `assert` needed?

Reply:
yes; 0 means that it was uninitialised.